.github/workflows/test.yaml

```diff
@@ -1,4 +1,4 @@
-name: Test Samples (k3d and EKS)
+name: Test Samples
```

Comment:
Maybe still keep k3d around for info on the cluster type being used?

```suggestion
name: Test Samples on k3d cluster
```

Reply:
fixed